configuration/connect-to-ms-exchange.md

```diff
@@ -0,0 +1,233 @@
+- SMTP localhost 1025 Autodetect Autodetect
+- Username: <YOUR-USERNAME>
+- Click on Re-Test and accept any certificate warning
+- (Checking the fingerprints!)
+- Click on Re-Test, then Done
+- Warning about unencrypted transfer from localhost
+- [x] I understand the risks
+
+
+## Setting up neomutt and offlineimap with Davmail
+See also: https://hobo.house/2015/09/09/take-control-of-your-email-with-mutt-offlineimap-notmuch/ assuming that you have installed and configured Davmail (see above)
+
+### Install packages in the TemplateVM
+install neomutt and offlineimap in the Template VM
+dom0:
+
+`qvm-run --auto t-mail gnome-terminal`
+
+in the Template VM:
+```
+sudo dnf install dnf-plugins-core
+sudo dnf copr enable flatcap/neomutt
+sudo dnf install neomutt dialog offlineimap git notmuch
+```
+additional tools for a good neomutt experience
+
+`sudo dnf -y install w3m qutebrowser mupdf`
+`shutdown -h now`
+
+restart AppVM if it was running before and you installed new packages in the Template VM
+dom0:
+
+`qvm-shutdown --wait --quiet my-workmail`
+`qvm-run --auto my-workmail gnome-terminal`
+
+Start & configure davmail (see above)
+
+`/opt/davmail/davmail.sh &`
+
+### Setting up neomutt
+Clone mutt-wizard
+
+`git clone https://github.com/LukeSmithxyz/mutt-wizard.git ~/.config/mutt`
+
+Launch mutt-wizard and choose add new account
+
+`.config/mutt/mutt-wizard.sh`
+
+- 1 Add an email account
+- start with adding something like user@qubes when asked for a GPG key
+- we will manually overwrite the config-file
+- fill out all entries, it's Ok if the last steps fails
+- verify if the account has been created:
+- 0 List all email accounts configured
+- 6 Exit this wizard
+
+Fix the configuration of offlineimap
+the configuration which has been created by mutt-wizard has to be changed in order to work with Davmail
+(Offlineimap will connect to davmail (localhost) and download emails)
+
+Create directory where offlineimap will store its mail
+
+`mkdir ~/.mail`
+
+Create a basic configuration file
+Change YOUR-USERNAME and YOUR-PASSWORD to your settings ;-)
+
+`nano ~/.offlineimaprc`
+
+Paste (overwrite the existing settings) the following lines into this file
+```
+[general]
+accounts = work
+starttls = no
+ssl = no
+sslcacertfile = /etc/ssl/certs/ca-certificates.crt
+
+[Account work]
+localrepository = work-local
+remoterepository = work-remote
+
+[Repository work-remote]
+type = IMAP
+remoteuser = YOUR-USERNAME
+sslcacerfile = /etc/ssl/cets/ca-certificates.crt
+remotepass = YOUR-PASSWORD
+remotehost = localhost
+remoteport = 1143
+sslcacertfile = /etc/ssl/certs/ca-certificates.crt
+ssl=no
+
+[Repository work-local]
+type = Maildir
+localfolders = ~/.mail/work
+```
+
+Create a link to the certfiles
+you could also put the correct path to the certfiles into .offlineimaprc but we're using the default paths which are also configures via mutt-wizard
+
+`cd /etc/ssl/certs/`
+`sudo ln -s ca-bundle.trust.crt ca-certificates.crt`  
```

Comment:
Since this is in AppVM, the change will not persist after restart. Better fix path in the config.

Reply:
Ok, but this step was made in the template VM.
Fixing the path might be a better solution.